deploy/kube-config/google/heapster-service.json

```diff
@@ -1,23 +0,0 @@
-{
```

Comment:
What about having it as `deploy/kube-config/heapster-service.json`, if it applies to all different configs?